packages/shared/src/normalizeProp.ts

```diff
@@ -24,6 +24,8 @@ export function normalizeStyle(
     return value
   } else if (isObject(value)) {
     return value
+  } else if (value === null || typeof value === 'undefined') {
+    return {}
```

Comment:
Can return `EMPTY_OBJECT` from `@vue/shared` here to avoid unnecessary allocation